src/index.js

```diff
@@ -1,2 +1,2 @@
-export Notification from './notification';
-export NotificationStack from './notificationStack';
+export { Notification } from './notification';
+export { NotificationStack } from './notificationStack';
```

Comment:
@joeyfigaro Can you remove this change from the PR? It's not related to the ES Lint and editor config changes and we can address this at a later time.